drizzlepac/outputimage.py

```diff
@@ -172,7 +172,8 @@ def set_units(self,units):
         self.units = units
 
     def writeFITS(self, template, sciarr, whtarr, ctxarr=None,
-                versions=None, overwrite=yes, blend=True, virtual=False):
+                versions=None, overwrite=yes, blend=True, virtual=False,
+                rules_file=None):
```

Comment:
I am just commenting everywhere that I see if a string should be set to None or an empty string.  None means the variable has not been set to a default value and "" means the default is an empty string.  There are just some places in code where None will not work for a string.  Pardon my repetition on this.